src/rust/engine/src/externs.rs

```diff
@@ -117,19 +120,28 @@ impl Externs {
     (self.store_list)(self.context, values_clone.as_ptr(), values_clone.len() as u64, merge)
   }
 
-  pub fn project(&self, value: &Value, field: &Field, type_id: &TypeId) -> Value {
-    (self.project)(self.context, value, field, type_id)
+  pub fn project(&self, value: &Value, field: &String, type_id: &TypeId) -> Value {
```

Comment:
Well, no. It would still be used in 99% of places. The only exception would be in the extern itself.